src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/TextBlock.cs

```diff
@@ -8,17 +8,17 @@
 
 namespace Microsoft.SqlTools.ServiceLayer.BatchParser
 {
-    internal class TextBlock
+    public class TextBlock
     {
         private readonly Parser parser;
         private readonly IEnumerable<Token> tokens;
 
         /// <summary>
         /// Constructor for the TextBlock class 
         /// </summary>
-        public TextBlock(Parser parser, Token token) : this(parser, new[] { token })
-        {
-        }
+        //public TextBlock(Parser parser, Token token) : this(parser, new[] { token })
```

Comment:
should this code be deleted instead of commented out?  also, it is a public method that presumably could be used by consumers of the library.  Is there any particular reason to remove this constructor?

Reply:
Reverted it